frontend/javascripts/oxalis/view/action-bar/merge_modal_view.tsx

```diff
@@ -290,20 +260,28 @@ class _MergeModalView extends PureComponent<Props, MergeModalViewState> {
                 onChange={this.handleChangeMergeExplorativeAnnotation}
               />
             </Form.Item>
-            <ButtonWithCheckbox
-              checkboxContent={mergeIntoActiveTracingCheckbox}
-              button={
+            <Form.Item>
+              <Tooltip title="Imports trees and tree groups directly into the currently opened annotation.">
                 <Button
                   type="primary"
-                  // @ts-expect-error ts-migrate(2322) FIXME: Type '"default"' is not assignable to type 'SizeTy... Remove this comment to see the full error message
-                  size="default"
                   disabled={this.state.selectedExplorativeAnnotation.length !== 24}
+                  onClick={this.handleImportExplorativeAnnotation}
+                >
+                  Import trees here
+                </Button>
+              </Tooltip>
+            </Form.Item>
+            <Form.Item>
+              <Tooltip title="Creates a new explorative annotation in your account with all merged contents of the current and selected annotations.">
+                <Button
+                  type="primary"
+                  disabled={this.state.selectedExplorativeAnnotation.length !== 24}
```

Comment:
I think, this line is incorrect :sweat_smile: I tracked it down to https://github.com/scalableminds/webknossos/commit/66527b720af208906ecb9f7e76f99626d5558bed, where it didn't really make sense, either. Probably length should be compared against 0?

Reply:
Ohhh, I see. The random 24 caught me off-guard and I couldn't make sense of it. how about extracting it into a `ANNOTATION_ID_LENGTH` constant?

Reply:
sorry for the confusion!

Reply:
No problem :) Sure, constant sounds good. Where would I put that? I’m always a bit lost in the tsx files to be honest :sweat_smile: